gop/gop.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025 The GoPlus Authors (goplus.org). All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gop
+
+import (
+	"github.com/goplus/mod/gopmod"
+	"github.com/goplus/mod/modfile"
+)
+
+func init() {
+	gopmod.SpxProject.Works = []*modfile.Class{{Ext: ".spx", Class: "SpriteImpl"}}
+}
+
+// SetSpxAutoImportedPackages sets the auto-imported packages for spx.
+func SetSpxAutoImportedPackages(pkgs map[string]string) {
```

Comment:
Don't specify spx, it's better to be general and identify spx by the Language id of lSP

Reply:
done https://github.com/goplus/goxlsw/compare/931b7759a829cf6328a5d5fff3347b76ee42c559..d62cb692a91205aa4c96e5877b9cd46d1b2799d2